bugsnag-android-core/src/main/java/com/bugsnag/android/BugsnagPluginInterface.kt

```diff
@@ -27,4 +27,7 @@ object BugsnagPluginInterface {
             null
         }
     }
+
+    fun createAnrEvent(exc: Throwable, client: Client) =
```

Comment:
We could move the method to another class such as `Event` as a static factory method but it would need to remain public if we want to avoid exposing the `Event` constructor. It felt best to move this to `BugsnagPluginInterface` or `NativeInterface` as these already contain several public methods that by convention should be used only within the notifier.

Reply:
Could the method be generalised and take a `HandledState`? The fact that we have ANR-specific code here is the part that sits less happily with me.

Reply:
That would expose the internally visible `HandledState` class as a parameter on a public function, which is not allowed in Kotlin. We could pass a `Severity` and `String` (for the severity reason) which would have the same effect.